public/components/presence-indicator/presence-indicators.js

```diff
@@ -20,12 +20,22 @@ function wfPresenceIndicatorsDirective ($rootScope, wfPresenceService,
                     $scope.presences = _.map(
                         _.uniqBy(currentState, (s) => { return s.clientId.person.email; }),
                         (pr) => {
-                            var person = pr.clientId.person;
-                            return { indicatorText:
-                                (person.firstName.charAt(0) + person.lastName.charAt(0)).toUpperCase(),
+                            const person = pr.clientId.person;
+
+                            const presenceObject = {
+                                indicatorText:
+                                    (person.firstName.charAt(0) + person.lastName.charAt(0)).toUpperCase(),
                                 longText: [person.firstName, person.lastName].join(" "),
-                                email: person.email,
-                                status: "present" };
+                                email: person.email
+                            };
+
+                            if(currentState[0].location === "body") {
+                                // the user is actively editing the body
+                                return {...presenceObject, ...{status: "present"}};
+                            } else {
+                                // the user is not editing the body, has clicked 'Save and close'
+                                return {...presenceObject, ...{status: "idle"}};
```

Comment:
Very much open to suggestions on a better name than "idle"

Reply:
Perhaps that's a good fit, yes - although notably, the Presence client doesn't appear to register any difference between editing and not editing furniture.

Reply:
> client doesn't appear to register any difference

This may be something we would want to address together with this bigger change that will allow multiple presence indicators in Liveblog blocks…

For now, this is great (as currently, editing furniture can happen concurrently to body editing).